dome-headend/dome-udp-server.ipv6.py

```diff
@@ -0,0 +1,72 @@
+import json
+import logging
+import socket
+
+log = logging.getLogger('dome-udp-server')
+
+FORMAT_CONS = '%(asctime)s %(name)-12s %(levelname)8s\t%(message)s'
+logging.basicConfig(level=logging.DEBUG, format=FORMAT_CONS)
+
+class Controller:
+	id = -1
+	num_leds = -1
+	start_index = -1
+	ip = ""
+
+class Led:
+	x = -1
+	y = -1
+	z = -1
+
+class DomeConfig:
+	controllers = [Controller()]
+	led_list = {Led()}
+
+	def __init__(self, config):
+		numControllers = len(config['controllers'])
+		numLeds = len(config['led_list'])
+		self.controllers = [Controller() for x in range(numControllers)]
+		self.led_list = {}
+
+		for controller in range(numControllers):
+			self.controllers[controller].id = config['controllers'][controller]['id']
+			self.controllers[controller].num_leds = config['controllers'][controller]['num_leds']
+			self.controllers[controller].start_index = config['controllers'][controller]['start_index']
+			self.controllers[controller].ip = config['controllers'][controller]['ip']
+
+		for led in range(numLeds):
+			newLed = Led()
+			newLed.x = config['led_list'][led]['x']
+			newLed.y = config['led_list'][led]['y']
+			newLed.z = config['led_list'][led]['z']
+			self.led_list[led] = newLed # TODO: Replace with ipv6
+		print self.led_list
+
+	def process_command(self, command, target):
+		log.debug("%r" % (command,))
+		log.debug("%r" % (target,))
```

Comment:
I'm confused - I assumed this file was just the IPv6 flaviour of dome-udp-server.py - but it doesn't appear to do the same thing?